packages/vc-export/src/exportToVerifiableCredential.spec.ts

```diff
@@ -14,9 +14,9 @@ import { Attestation, AttestedClaim, CType } from '@kiltprotocol/core'
 import { DidUtils } from '@kiltprotocol/did'
 import { Crypto } from '@kiltprotocol/utils'
 import { DocumentLoader } from 'jsonld-signatures'
-import toVC from './exportToVerifiableCredential'
-import verificationUtils, { AttestationStatus } from './verificationUtils'
-import claimerUtils, { makePresentation } from './presentationUtils'
+import * as toVC from './exportToVerifiableCredential'
+import * as verificationUtils from './verificationUtils'
+import * as claimerUtils from './presentationUtils'
```

Comment:
S: maybe the variable name could be `presentationUtils`, too

Reply:
https://github.com/KILTprotocol/sdk-js/pull/434/commits/da5a033c78a91a4087f95779403f0ae429bb03e2